signals/sender/sender_legitimate_domain_negation.yml

```diff
@@ -0,0 +1,27 @@
+name: "Sender: Legitimate message senders"
```

Comment:
What's the idea here, to fire on secure mailers? Right now, it looks like this is just "not a known secure mailer".

Reply:
yeah. i need to `or` these and remove the `not` logic